powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java

```diff
@@ -804,31 +774,6 @@ private TokenInfo createToken() throws InvalidKeySpecException, CryptoProviderEx
         return tokenInfo;
     }
 
-    private EciesParameters getEciesRequestParameters(String version, byte[] associatedData) throws CryptoProviderException {
-        if ("3.2".equals(version)) {
-            final Long timestamp = new Date().getTime();
-            final byte[] nonceBytes = keyGenerator.generateRandomBytes(16);
-            return EciesParameters.builder().nonce(nonceBytes).associatedData(associatedData).timestamp(timestamp).build();
-        }
-        if ("3.1".equals(version)) {
-            final byte[] nonceBytes = keyGenerator.generateRandomBytes(16);
-            return EciesParameters.builder().nonce(nonceBytes).build();
-        }
-        return EciesParameters.builder().build();
-    }
-
-    private EciesParameters getEciesResponseParameters(String version, byte[] associatedData, byte[] requestNonce) throws CryptoProviderException {
-        if ("3.2".equals(version)) {
```

Comment:
So _protocolVersion_ 3.2 will be tested later on, right?

Reply:
Yes, we already use a separate package for that (see `v31` in the path). It would be mess to put all tests together and make all required exceptions.